plugins/spi/SPIOutputTest.cpp

```diff
@@ -738,3 +739,233 @@ void SPIOutputTest::testCombinedAPA102Control() {
   // check if the output writes are 1
   OLA_ASSERT_EQ(1u, backend.Writes(1));
 }
+
+
+/**
+ * Test DMX writes in the individual APA102 Pixel Brightness mode.
+ */
+void SPIOutputTest::testIndividualAPA102ControlPixelBrightness() {
+  // personality 7= Individual APA102
```

Comment:
This should be personality 9, 7 is individual no PB.

Reply:
Thinking about trying to fix this, I don't know what @nomis52 thinks, but to me it would be good if we had enums for the personalties, as then this problem, and the switch case to decide what mode we're in can be neater.

If we list the enums, then we can do
personalities.insert(personalities.begin() + PERS_WS2801_INDIVIDUAL, Personality("WS2801 Individual"));

Then switch/run tests based on PERS_WS2801_INDIVIDUAL.

Or we could build an array and convert it to a vector.